CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - The `WithoutTimestamps` option to `go.opentelemetry.io/otel/exporters/stdout/stdoutmetric` to sets all timestamps to zero. (#3828)
 - The new `Exemplar` type is added to `go.opentelemetry.io/otel/sdk/metric/metricdata`.
   Both the `DataPoint` and `HistogramDataPoint` types from that package have a new field of `Exemplars` containing the sampled exemplars for their timeseries. (#3849)
+- The internal logging changes the verbosity level of info to `V(4)`, the verbosity level of debug to `V(8)`, and introduces a warning level equal to `V(1)`. (#3900)
```

Comment:
I suggest moving it to `Changed` section (instead of `Added`)

Reply:
yeap agreed, actually, I've split it into two parts now, one line for _changed_ one line for _added_ section